nbconvert/exporters/templateexporter.py

```diff
@@ -50,6 +50,7 @@
         'prevent_list_blocks': filters.prevent_list_blocks,
         'get_metadata': filters.get_metadata,
         'convert_pandoc': filters.convert_pandoc,
+        'wrapped_convert_link': wrapped_convert_link,
```

Comment:
hmmm, I don't think that this idiom works exactly as is (also I think this is not quite how bookbook does it) I'll work on it.